### ISSUE TITLE
[Bug] AddProjectHookOptions.BuildEvents should be renamed to JobEvents

### ISSUE THREAD

**Issue by sindrepm** [xanzy/go-gitlab#189]
I am getting an error when trying to add a new project hook on Gitlab 9.3.8. Turns out that the `build_events` parameter has been renamed to `job_events`:

https://docs.gitlab.com/ce/api/projects.html#add-project-hook
```
Attribute | Type | Required | Description
job_events | boolean | no | Trigger hook on job events
```
Head of `master` branch still uses `build_events` parameter. See https://github.com/xanzy/go-gitlab/blob/master/projects.go#L745

Is this something that can be fixed as currently it is not possible to add project hooks with this client on newer Gitlab installations.

**Comment by svanharmelen** [comment]
Currently we try to keep master backwards compatible and are working on full API V4 support is a feature branch (f-api-v4).

For now I would suggest you use that branch instead of master. Hopefully after the vacation times are over we round up the V4 support and move that to master.

Current master will then become a maintenance branch... Hope this helps. On holiday myself as well now, so not around a laptop or desktop.

**Comment by sindrepm** [comment]
@svanharmelen Great to hear that API V4 support is finally coming as it has been the preferred version since GitLab 9.0 and API V3 will be unsupported from GitLab 9.5. I already had a look at the `f-api-v4` feature branch but it doesn't seem like this change is in yet, so I created a local fork for now.

**Comment by svanharmelen** [comment]
Check... Well of course PR's are always welcome :wink:

**Comment by sindrepm** [comment]
@svanharmelen Now that f-api-v4 has been merged to master, this part of the API is not working at the moment, so I went ahead and implemented the changes in PR #207 making it compatible with V4.
